setup.py

```diff
@@ -1,39 +1,27 @@
-"""qiskit_sphinx_theme
-
-A Sphinx theme for Qiskit that is based on the
-Pytorch Sphinx theme.
-"""
 from setuptools import setup
 
-DOCLINES = __doc__.split('\n')
-DESCRIPTION = DOCLINES[0]
-LONG_DESCRIPTION = "\n".join(DOCLINES[2:])
-
 setup(
-    name = 'qiskit_sphinx_theme',
-    version = '1.11.0rc1',
+    name='qiskit_sphinx_theme',
+    version='1.11.0rc1',
     author="Qiskit Development Team",
     author_email="hello@qiskit.org",
     url="https://github.com/Qiskit/qiskit_sphinx_theme",
-    description=DESCRIPTION,
-    long_description=LONG_DESCRIPTION,
```

Comment:
We don't need to set `long_description`. The former `description` of only `qiskit_sphinx_theme` wasn't very informative.